ci/snap/init/snapcraft.yaml

```diff
@@ -6,7 +6,7 @@ description: |
 grade: stable
 confinement: strict
 base: core22
-issues: https://github.com/canonical/ubuntu-desktop-provision/issues
+issues: https://bugs.launchpad.net/ubuntu-desktop-provision/+filebug
 contact: https://github.com/canonical/ubuntu-desktop-provision/issues
```

Comment:
See above, shouldn't we change this to launchpad as well?

Reply:
See discussion above.